endoreg_db/data/pdf_type/data.yaml

```
@@ -1,29 +1,60 @@
+    examiner_info_line: "ukw-examination-generic-examiner-info-line"
     cut_off_above_lines: [
-      "ukw-histology-generic-patient-info-line"
+      "ukw-examination-generic-anamnese-info-line",
+      "ukw-examination-generic-endoscope-info-line",
+      "ukw-examination-generic-examiner-info-line",
     ]
     cut_off_below_lines: [
-      "ukw-endoscopy-histology-cut-off-below-line-flag-01"
-    ]
+      "rkh-histology-generic-cut-off-below-line"
+    ]
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)